source/libnormaliz/cone.cpp

```diff
@@ -6970,8 +6970,10 @@ void Cone<Integer>::make_face_lattice(const ConeProperties& ToCompute){
             if(skip_remaining)
                 continue;
             
-           for(; kkk > Fpos; ++Fpos, ++F);
-           for(; kkk < Fpos; --Fpos, --F) ;
+            for(; kkk > Fpos; ++Fpos, ++F)
+                ;
+            for(; kkk < Fpos; --Fpos, --F)
+                ;
```

Comment:
This hint (putting the semicolon on a new line) tells clang and gcc that the body of the loop is intentionally empty.

Reply:
Will follow this suggestion everywhere.